### PR TITLE
Add datatable w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by AilurusUmbra**
Pull request for issue: #10718 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):datatable
 * Official website (optional, not the repository):
 * NPM package url (optional): none
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 76
   - Count of forks: 24

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: node_modules/datatable/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
```
Comment (PeterDaveHello) [inline node_modules/datatable/package.json]: This file should be removed.

**Comment by PeterDaveHello** [comment]
@AilurusUmbra  can you please fill the comment template for the library info?

**Comment by PeterDaveHello** [comment]
I think the commit 333dab6 should be removed here? Thanks.

**Comment by AilurusUmbra** [comment]
@sufu3 
those 2 commits are squashed!

**Comment by PeterDaveHello** [comment]
@AilurusUmbra please please fill the comment template for the library info next time, thanks.

**Pull request merged**